assets/css/style.scss

```diff
@@ -33,3 +33,21 @@
   justify-content: center;
 }
 
+.image-with-quote {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+
+  blockquote {
+    height: fit-content;
+  }
+}
+
+.about-image {
+  height: 200px;
```

Comment:
This looks bad on mobile :( Can we maintain the aspect ratio? I think that would solve it...